### PR TITLE
remove okteto push

### PR DESCRIPTION
**Pull request by jLopezbarb**
# Proposed changes

Fixes DEV-426

This PR removes all the code related to okteto push:
- CI
- make file
- serialisation
- gh action tests
- okteto push command
- analytics
 

## CLI Quality Reminders 🔧

For both authors and reviewers:

- Scrutinize for potential regressions
- Ensure key automated tests are in place
- Build the CLI and test using the validation steps
- Assess Developer Experience impact (log messages, performances, etc)
- If too broad, consider breaking into smaller PRs
- Adhere to our [code style](https://github.com/okteto/okteto/blob/master/docs/code-style.md) and [code review](https://github.com/okteto/okteto/blob/master/docs/code-review.md) guidelines

### CODE REVIEW

**Comment by andreafalzetti** [comment]
@jLopezbarb can you please run the e2e test with the PR? I think it's a good practice with all the 3.0 related-work

File: cmd/utils/dev.go [hunk 1]
```diff
@@ -116,32 +68,6 @@ func LoadManifestRc(dev *model.Dev) error {
...
 
-// DeprecatedLoadManifestOrDefault loads an okteto manifest or a default one if does not exist
-// Deprecatd. It should only be used by `push` command that will be deleted on next major version. No new usages should be added
-func DeprecatedLoadManifestOrDefault(devPath, name string, fs afero.Fs) (*model.Manifest, error) {
-	dev, err := DeprecatedLoadManifest(devPath, fs)
-	if err == nil {
-		return dev, nil
-	}
-
-	if oktetoErrors.IsNotExist(err) && len(name) > 0 {
```
Comment (andreafalzetti) [inline cmd/utils/dev.go:127]: `oktetoErrors.IsNotExist` can also be removed, it's not used anywhere else

File: cmd/utils/dev.go [hunk 2]
```diff
@@ -48,51 +45,6 @@ const (
...
-		if devPath == DefaultManifest {
-			if filesystem.FileExists(secondaryManifest) {
-				return DeprecatedLoadManifest(secondaryManifest, fs)
-			}
-		}
-
-		return nil, fmt.Errorf("'%s' does not exist. Generate it by executing 'okteto init'", devPath)
-	}
-
-	manifest, err := model.Get(devPath, fs)
```
Comment (andreafalzetti) [inline cmd/utils/dev.go:64]: I think we can also delete `model.Get` - please double check

**Review by andreafalzetti — COMMENTED** [review]
Remember to deprecate the GHA as well as part of this ticket

**Review by andreafalzetti — APPROVED** [review]

**Review by ifbyol — APPROVED** [review]

**Pull request merged**